/**
 * Defines index with each message
 * 
 * User 100
 * - User: 101
 * - Teacher: 103
 * - RequestLogin: 105
 * - ResponseLogin: 107
 * - UserInit: 109
 * - LoginStatus: 111
 * Class 200
 * - Category: 201
 * - ClassroomInfo: 203
 * - RequestViewCategoryDetail: 205
 * - ClassroomInfoOfCategory: 207
 * Class Online 300
 * - TeacherOpenClass = 301
 * - ClassOnlineAction = 303
 * Board 400
 * - ChangeFontSetting = 401
 * - DrawLine: 402
 * - Erase: 403
 */

package METRO.CRO.MESSAGES;

message IpcMessage {
	optional int64 id = 1;
	optional bytes payload_data = 2;
}
